fearless/Modules/Staking/ValidatorInfo/ValidatorInfoViewModel/ValidatorInfoViewModelFactory.swift

```diff
@@ -0,0 +1,163 @@
+        }
+
+        let estimatedRewardLocalizedContent: LocalizableResource<TitleWithSubtitleViewModel> =
+            LocalizableResource { locale in
+                let percentageFormatter = NumberFormatter.percentAPY.localizableResource().value(for: locale)
+
+                let title = R.string.localizable
+                    .stakingValidatorEstimatedReward(preferredLanguages: locale.rLanguages)
+
+                let subtitle = percentageFormatter
+                    .string(from: stakeInfo.stakeReturn as NSNumber) ?? ""
+
+                return TitleWithSubtitleViewModel(title: title,
+                                                  subtitle: subtitle)
+        }
+
+        stakingViewModel.append((rowType: .totalStake, content: totalStakeLocalizedContent))
+        stakingViewModel.append((rowType: .nominators, content: nominatorsLocalizedContent))
+        stakingViewModel.append((rowType: .estimatedReward, content: estimatedRewardLocalizedContent))
+
+        guard stakingViewModel.count > 0 else { return nil }
+
+        return stakingViewModel
+    }
+
+    private func generateIdentityViewModel(
+        from validatorInfo: ValidatorInfoProtocol) -> [ValidatorInfoViewController.Row]? {
+        guard let identity = validatorInfo.identity else { return nil }
+
+        var identityViewModel: [ValidatorInfoViewController.Row] = []
+
+        if let legal = identity.legal {
+            let content: LocalizableResource<TitleWithSubtitleViewModel> = LocalizableResource { locale in
+                let title = R.string.localizable.identityLegalNameTitle(preferredLanguages: locale.rLanguages)
+                return TitleWithSubtitleViewModel(title: title, subtitle: legal)
+            }
+
+            identityViewModel.append((rowType: .legalName, content: content))
+        }
+
+        if let email = identity.email {
+            let content: LocalizableResource<TitleWithSubtitleViewModel> =
+                LocalizableResource { locale in
+                    let title = R.string.localizable.identityEmailTitle(preferredLanguages: locale.rLanguages)
+                    return TitleWithSubtitleViewModel(title: title, subtitle: email)
+            }
+
+            identityViewModel.append((rowType: .email, content: content))
+        }
+
+        if let web = identity.web {
+            let content: LocalizableResource<TitleWithSubtitleViewModel> =
+                LocalizableResource { locale in
+                    let title = R.string.localizable.identityWebTitle(preferredLanguages: locale.rLanguages)
+                    return TitleWithSubtitleViewModel(title: title, subtitle: web)
+            }
+
+            identityViewModel.append((rowType: .web, content: content))
+        }
+
+        if let twitter = identity.twitter {
+            let content: LocalizableResource<TitleWithSubtitleViewModel> = LocalizableResource { _ in
+                    return TitleWithSubtitleViewModel(title: "Twitter", subtitle: twitter)
```

Comment:
Why riot better than twitter? Let's add localization as we did for riot

Reply:
It's because there wasn't any string in Lokilise representing twitter. Personally I don't discriminate (: Let's do it